pulp_ansible/app/galaxy/v3/views.py

```diff
@@ -925,6 +985,8 @@ class CollectionImportViewSet(
     queryset = CollectionImport.objects.prefetch_related("task").all()
     serializer_class = CollectionImportDetailSerializer
 
+    queryset_filtering_required_permission = "ansible.view_ansiblerepository"
+
```

Comment:
Since this viewset is not a `NamedModelViewSet`, I don't think `queryset_filtering_required_permission` does anything. If you want to do some queryset filtering I would just define a `get_queryset` method for this viewset and perform the filter there. See the last two paragraphs here: https://docs.pulpproject.org/pulpcore/plugin_dev/plugin-writer/concepts/rbac/queryset_scoping.html